deploy/uninstall-driver.sh

```diff
@@ -34,6 +38,8 @@ if [ $ver != "master" ]; then
 fi
 
 echo "Uninstalling NFS driver, version: $ver ..."
+# this keeps VolumeSnapshot CRDs untouched
+kubectl -n kube-system kustomize "$snap_controller_repo" | kubectl delete --ignore-not-found -f -
```

Comment:
shall we use current csi-snapshot-controller.yaml and CRDs to install snapshot controller?

Reply:
tbh, I would love to tackle https://github.com/kubernetes-csi/external-snapshotter/issues/812 next, which would ideally lead to CSI drivers using helm dependencies https://helm.sh/docs/helm/helm_dependency/ and `kubectl kustomize` to install any external-snapshotter resources. This and other CSI drivers, could then track their dependencies on a few lines by referencing upstream sources, instead of copy/pasting external-snapshotter resources in `./deploy` and `./charts`. And this would simplify version bumps for both installation options and keep things up to date, imho.

But I will defer to your expert opinion.